velox/common/base/VeloxException.cpp

```diff
@@ -258,10 +258,10 @@ void VeloxException::State::finalize() const {
     elaborateMessage += "Stack trace has been disabled.";
     if (exceptionType == VeloxException::Type::kSystem) {
       elaborateMessage +=
-          "Use --velox_exception_system_stacktrace=true to enable it.\n";
+          " Set gflag velox_exception_system_stacktrace_enabled=true to enable it.\n";
```

Comment:
Are you changing the flag names? That makes sense, but let's keep the original language of "Use xxx"

Reply:
Thanks, already fixed.